app/controllers/posts_controller.rb

```diff
@@ -10,26 +10,35 @@ def show
     @posts = Post.all
     @post = Post.find params[:id]
     @post_id = @posts.index(@post) + 1
+    @user = @post.author
```

Comment:
- [ ] according to the requirements your application should use CanCanCan for Post authorization in Post Controller

Reply:
Resolved

Reply:
well resolved! ✔️